The YES and NO Objective-C macros clash with enum values of the same name in
DisposeAfterUse class.

--- backends/platform/sdl/macosx/macosx-window.h.orig
+++ backends/platform/sdl/macosx/macosx-window.h
@@ -25,6 +25,7 @@
 
 #ifdef MACOSX
 
+#include "common/ptr.h"
 #include "backends/platform/sdl/sdl-window.h"
 
 class SdlWindow_MacOSX final : public SdlWindow {
